src/dbnode/storage/index.go

```diff
@@ -541,6 +554,10 @@ func (i *nsIndex) reportStats() error {
 			return err
 		}
 	}
+	// Active block should always be open.
```

Comment:
Is it worth checking `activeBlock.IsOpen` here?

Reply:
We don't close it until it's explicitly closed in `Close()`.

Maybe the comment should read "Active block is always open until index closed, at which point report stats is not run, therefore no check required".